<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comporator</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QRadioButton {
	margin-left: 50px;
	height: 60px;
}
QRadioButton#radioButton {
	margin-top: 20px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="page_1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioOpt1">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Create LCR from all Supplier Sheets</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioOpt2">
          <property name="text">
           <string>Customize</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>60</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="destlabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Enter the Destination:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="destEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="codeLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Enter the Dial Code:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="codeEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="rateLabel">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Enter the Rate:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="rateEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Upload customer rate sheet</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="page1NextBtn">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Add sheets for creating LCR:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="filesList">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item row="1" column="2">
           <widget class="QPushButton" name="p2DellAllBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Delete all</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="page2NextBtn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Next</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btnAddFiles">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Add files</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnDelFiles">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Delete selected</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="page2BackBtn">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Back</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="clrTable">
          <column>
           <property name="text">
            <string>Destination</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Codes</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Files...</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="clrErrorLabel">
          <property name="text">
           <string>Errors occurred in the following files:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="clrErrors">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QListWidget" name="clrUndefSheetslist">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>80</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <widget class="QPushButton" name="page3GoBackBtn">
            <property name="minimumSize">
             <size>
              <width>152</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>&lt; Back to select option</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="page3ShowErrorsBtn">
            <property name="minimumSize">
             <size>
              <width>152</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Show sheets errors</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="page3SetAliasesBtn">
            <property name="minimumSize">
             <size>
              <width>152</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Set aliases</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>180</y>
          <width>741</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>100</y>
          <width>82</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>943</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>radioOpt1</tabstop>
  <tabstop>codeEdit</tabstop>
  <tabstop>radioOpt2</tabstop>
  <tabstop>rateEdit</tabstop>
  <tabstop>page1NextBtn</tabstop>
  <tabstop>filesList</tabstop>
  <tabstop>destEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioOpt2</sender>
   <signal>toggled(bool)</signal>
   <receiver>destlabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>170</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioOpt2</sender>
   <signal>toggled(bool)</signal>
   <receiver>codeLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioOpt2</sender>
   <signal>toggled(bool)</signal>
   <receiver>rateLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>157</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioOpt2</sender>
   <signal>toggled(bool)</signal>
   <receiver>destEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioOpt2</sender>
   <signal>toggled(bool)</signal>
   <receiver>codeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioOpt2</sender>
   <signal>toggled(bool)</signal>
   <receiver>rateEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
